bin/chef-cluster

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+#
+# Copyright 2017, SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmd=$1
+workercount=${2:-auto}
+vhostconfig=/etc/apache2/vhosts.d/chef-cluster.conf
+serverconfig=/etc/sysconfig/chef-server
+webroot=/srv/www/chef-cluster
+
+baseport=4001
+
+balancerstart=BEGIN-BALANCER-MEMBER
+balancerend=END-BALANCER-MEMBER
+
+usage() {
+  echo "usage: chef-cluster <enable|disable> [<workercount>|auto]"
+  exit 1
+}
+
+case $cmd in
+  enable)
+    [ "$workercount" == "auto" ] && workercount=$(grep -c ^processor /proc/cpuinfo)
+    [ "$workercount" -ge 2 ] || { echo "ERROR: workercount must be >= 2"; exit 2; }
+
+    # apache configuration
+    mkdir -p "$webroot"
+    a2enmod -q lbmethod_byrequests || a2enmod lbmethod_byrequests
```

Comment:
To be super-safe, shouldn't we also enable proxy_balancer? (In case the rest of the crowbar setup stops enabling it at some point)

Reply:
done